### PR TITLE
[feature] piv: partial removal

### PR DESCRIPTION
**Pull request by patochectp**
ref: ND-1084

### CODE REVIEW

File: tests/integration/piv_test.py [hunk 1]
```diff
@@ -305,6 +305,14 @@ def test_piv_event_priority(mock_rabbitmq):
         assert db_trip_removal.message == "Indisponibilité d'un matériel"
         # full trip removal : no stop_time to precise
         assert len(db_trip_removal.stop_time_updates) == 0
+
+        for s in db_trip_removal.stop_time_updates[0:16]:
+            assert s.arrival_status == "none"
+            assert s.departure_status == "none"
+            assert s.message is None
+            assert s.arrival_status == "delete"
+            assert s.departure_status == "delete"
```
Comment (woshilapin) [inline tests/integration/piv_test.py:314]: I do not understand how is it possible that both
```python
             assert s.arrival_status == "none"
```
and
```python
             assert s.arrival_status == "delete"
```
can be true :open_mouth:

**Review by woshilapin — APPROVED** [review]

File: tests/integration/piv_test.py (hunk 1 above)
Comment (patochectp) [reply tests/integration/piv_test.py]: db_trip_partial_removed.stop_time_updates len = 0 so the for loop is not executed

**Review by pbougue — APPROVED** [review]
Apart from the question it's OK, so we might also merge and add another PR if needed.

File: kirin/piv/model_maker.py [hunk 2]
```diff
@@ -79,7 +79,8 @@
     TripEffect.UNDEFINED: maxint,
 }
 
-MANAGED_EVENTS = ["RETARD", "SUPPRESSION"]
+MANAGED_EVENTS = ["RETARD", "SUPPRESSION", "MODIFICATION_DESSERTE_SUPPRIMEE"]
```
[the remark this answers is not in the document]
Comment (pbougue) [reply kirin/piv/model_maker.py:82]: To be done in a separate ticket, but yes, it's the same behaviour.

**Pull request merged**